modules/eventsource/manager_run.go

```diff
@@ -9,13 +9,13 @@ import (
 
 	activities_model "code.gitea.io/gitea/models/activities"
 	issues_model "code.gitea.io/gitea/models/issues"
-	"code.gitea.io/gitea/modules/convert"
 	"code.gitea.io/gitea/modules/graceful"
 	"code.gitea.io/gitea/modules/json"
 	"code.gitea.io/gitea/modules/log"
 	"code.gitea.io/gitea/modules/process"
 	"code.gitea.io/gitea/modules/setting"
 	"code.gitea.io/gitea/modules/timeutil"
+	"code.gitea.io/gitea/services/convert"
```

Comment:
After a quick search, this package does not seem to be called by `modules` or `models`.
So please move it to `services` as well, then I'll approve.

Reply:
You talk about the `eventsource`? Why not move the packages one by one? This PR is currently focused on the `convert` package.

Reply:
Okay, if you want to move each package separately…

Reply:
I think the toughest one will be the `context` module which may require more code changes. So it may be easier to review if not many packages are moved in a single PR. But sure, we can move other packages in one PR if the only changes are the imports. 👍